codalab/lib/bundle_cli.py

```diff
@@ -1924,14 +1926,21 @@ def do_search_command(self, args):
             params={'worksheet': worksheet_uuid, 'keywords': args.keywords, 'include': ['owner']},
         )
 
+        print('Note: at most 10 results will be shown.')
+
         # Print direct numeric result
         if 'meta' in bundles:
+            if len(bundles['meta']['result']) == 0:
```

Comment:
I don't think we need this here

Reply:
I see. I thought we also wanted to print that 10 results at most note when no aggregated information is returned. But I agree with you that it is needed here since it is aggregated result like count, etc.